Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift

```diff
@@ -0,0 +1,143 @@
+                    }
+                }
+                
+                // A series of buttons for general language strategies.
+                ForEach(
+                    [
+                        // Use the default language setting for the basemap style.
+                        BasemapStyleLanguage.default,
+                        // Use the global language (English) for basemap labels.
+                        BasemapStyleLanguage.global,
+                        // Uses country-local language for basemap labels.
+                        BasemapStyleLanguage.local,
+                        // Use the system locale language for basemap labels.
+                        BasemapStyleLanguage.applicationLocale
+                    ],
+                    id: \.label
+                ) { basemapStyleLanguage in
+                    Button {
+                        selectedLanguage = basemapStyleLanguage
+                    } label: {
+                        HStack {
+                            Text(basemapStyleLanguage.label)
+                            Spacer()
+                            if selectedLanguage == basemapStyleLanguage {
+                                Image(systemName: "checkmark")
+                                    .foregroundColor(.accentColor)
+                            }
+                        }
+                        .contentShape(Rectangle())
+                    }
+                    .buttonStyle(.plain)
+                }
+            }
+        }
+    }
+}
+
+private extension ConfigureBasemapStyleParametersView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with OpenStreetMap light gray basemap.
+        let map: Map = {
+            let map = Map(
+                basemap: Basemap(
+                    // An OpenStreetMap basemap style is used to support localization.
+                    style: .osmLightGray,
+                    // Set the language strategy to global to use English.
+                    parameters: BasemapStyleParameters(language: .global)
+                )
+            )
+            // Start with a viewpoint over Bulgaria, Greece, and Turkey.
+            // They use three different alphabets: Cyrillic, Greek, and Latin, respectively.
+            map.initialViewpoint = Viewpoint(center: Point(x: 3_000_000, y: 4_500_000), scale: 1e7)
+            return map
+        }()
+        
+        /// The language label and Esri language code for the basemap style parameters.
+        let languages: KeyValuePairs<String, String> = [
+            "🇧🇬 Bulgarian": "bg",
+            "🇬🇷 Greek": "el",
+            "🇹🇷 Turkish": "tr"
+        ]
+        
+        /// Sets the basemap style parameter with a language strategy.
+        /// - Parameter language: The language setting for the basemap.
+        func setBasemapStyleParameters(language: BasemapStyleLanguage) {
```

Comment:
```suggestion
        func setBasemapLanguage(_ language: BasemapStyleLanguage) {
```

Not a big deal but this might describe what it does better.